app/ts/components/App.tsx

```diff
@@ -23,3 +26,10 @@ const Transaction = () => {
 		</div>
 	)
 }
+
+function useSplashEffect() {
+	useEffect(() => {
+		const element = document.querySelector('.splash-screen')
+		element?.classList.add('splash-screen--off')
+	}, [])
+}
```

Comment:
Should this be renamed to `removeSplash()` or something, since I believe that is what it is trying to do?

Reply:
this was indeed named before figuring the mechanism for animation. i'm fine renaming it tho i've seen common use of functions prefixed with `use` for hooks, if that's something that we should also adapt? some variations i can also think of:

- useSplashScreenExit
- useSplashScreenTransitionEffect
- useTransitionFromSplash
- useExitSplashScreenEffect

or any of which without the `use` prefix. does these seem fit to you?

Reply:
added a `classList` check before injecting the classname